shecoded.html

```diff
@@ -76,7 +76,7 @@
 
     .she-coded-hero-banner__logo {
```

Comment:
@benhalpern with this just double check that it doesn't mess up the resolution on other screen sizes